Sources/ArcGISToolkit/Components/Search/SearchSource.swift

```diff
@@ -15,7 +15,7 @@
 import ArcGIS
 
 /// Defines the contract for a search result provider.
-public protocol SearchSource {
+@MainActor public protocol SearchSource {
```

Comment:
```suggestion
@MainActor
@preconcurrency
public protocol SearchSource {
```

Reply:
Why is `@preconcurrency` needed here?

Reply:
Because a new concurrency constraint is being added. Without it, if a consumer hasn't adopted complete strict concurrency checking, they will get a warnings (Swift 5 language mode) or an error (Swift 6 language mode).

Reply:
Makes sense, thank you.